Added Timezone Resource (#5758)
chef/inspec: docs-chef-io/content/inspec/resources/timezone.md

+++
title = "timezone resource"
draft = false
gh_repo = "inspec"
platform = "linux"

[menu]
  [menu.inspec]
    title = "timezone"
    identifier = "inspec/resources/os/timezone.md timezone resource"
    parent = "inspec/resources/os"
+++

Use the `timezone` Chef InSpec audit resource to test timezone configurations of the system.

## Availability

### Installation

This resource is distributed along with Chef InSpec itself. You can use it automatically.

## Syntax

> A `timezone` resource fetches the timezone configurations of the system and compares the output with the test:

Reviewer: ```suggestion
A `timezone` resource block fetches the time zone configuration of a system and compares the output with the test:
```

    describe timezone do
      its('property') { should eq 'expected value' }
    end

> where

Reviewer: ```suggestion
where:
```

- `'property'` is one of `identifier` , `name` and `time_offset`
- `'expected value'` tests the output of the command run on the system versus the expected output stated in the test

For example:

    describe timezone do
      its('identifier') { should eq 'Asia/Kolkata' }
      its('name') { should eq 'IST' }
      its('time_offset') { should eq '+0530' }
    end

## Properties

### identifier

The `identifier` property returns the location identifier of the timezone.

> An example of checking the **identifier** for India Time :

Reviewer: ```suggestion
An example of checking the **identifier** for the Asia/Kolkata time zone name:
```

>     its('identifier') { should eq 'Asia/Kolkata }

Reviewer: ```suggestion
    its('identifier') { should eq 'Asia/Kolkata' }
```

### name

> The `name` property returns the name of the timezone.

Reviewer: ```suggestion
The `name` property returns the time zone abbreviation.
```

An example of checking the **name** of India Time :

>     its('name') { should eq 'IST' }
> 

Reviewer: ```suggestion
    its('name') { should eq 'IST' }

{{< note >}}

Several time zones share the same time zone abbreviation. Use one of the other properties to verify a specific time zone with a common abbreviation.

{{< /note >}}

```

### time_offset

The `time_offset` property returns the identifier of a time offset from UTC (Coordinated Universal Time).

> An example of checking the **time_offset** of India Time:

Reviewer: ```suggestion
An example of checking if the **time_offset** is UTC+05:30:
```

+++
title = "timezone resource"
draft = false
gh_repo = "inspec"
platform = "linux"

[menu]
  [menu.inspec]
    title = "timezone"
    identifier = "inspec/resources/os/timezone.md timezone resource"
    parent = "inspec/resources/os"
+++

Use the `timezone` Chef InSpec audit resource to test timezone configurations of the system.

## Availability

### Installation

This resource is distributed along with Chef InSpec itself. You can use it automatically.

### Version

> This resource first became available in v1.0.0 of InSpec.

Reviewer 2: Uh, current version is 4.53 ....

Author: Geez, my bad. Will change this. Thanks!
